go.mod

```diff
@@ -26,7 +26,7 @@ require (
 	github.com/huandu/xstrings v1.2.0 // indirect
 	github.com/imdario/mergo v0.3.7
 	github.com/julienschmidt/httprouter v1.2.0
-	github.com/linkerd/linkerd2-proxy-api v0.1.13
+	github.com/linkerd/linkerd2-proxy-api v0.1.14-0.20200928162511-7fcd01ab736d
```

Comment:
do we need the pseudoversion looking thing on the version here?  can we just use v0.1.14?

Reply:
Yep 1.14 was released now so I'll change this